torch/nn/quantized/modules/conv.py

```diff
@@ -80,8 +80,8 @@ def __init__(self, in_channels, out_channels, kernel_size, stride=1,
         # Initialize as NCHW. set_weight will internally transpose to
         # NHWC
         qweight = torch._empty_affine_quantized(
-            [out_channels, in_channels // self.groups, kernel_size[0],
-             kernel_size[1]],
+            [out_channels, in_channels // self.groups, self.kernel_size[0],
+                self.kernel_size[1]],
```

Comment:
does this fail lint?

Reply:
it passes lint (on the diff); i think that it wants it to be indented